src/lib/apiTypesV2/Entity.cpp

```diff
@@ -237,12 +248,23 @@ void Entity::present(const std::string& indent)
 *
 * Entity::fill - 
 */
-void Entity::fill(const std::string& _id, const std::string& _type, const std::string& _isPattern, ContextAttributeVector* aVec)
+void Entity::fill
+(
+  const std::string&       _id,
+  const std::string&       _type,
+  const std::string&       _isPattern,
+  ContextAttributeVector*  aVec,
+  double                   _creDate,
+  double                   _modDate
```

Comment:
I was thinking maybe it was to have decimals of a second ...
If that is not the case, let's open an issue. Doubles give rounding problems that integers don't have.

NTC